packages/flutter/lib/fix_data/fix_cupertino.yaml

```
@@ -295,4 +295,50 @@ transforms:
     - kind: 'rename'
       newName: 'buildOverscrollIndicator'
 
+  # Changes made in https://github.com/flutter/flutter/pull/151367
+  - title: "Rename to 'activeTrackColor'"
+    date: 2024-07-15
+    element:
+      uris: [ 'cupertino.dart' ]
+      getter: 'activeColor'
```

```suggestion
      getter: activeColor
```
Looks like for `getter` you don't use quotation marks. Sorry, dart fix is really cool, but somewhat finicky.

---

I took example from fix_color_scheme.yaml:432, there we are using quotation marks

---

Anyways removing quotation marks, doesn't solve the problem

---

Let me play around with this locally. I feel like the answer is going to turn out to be obvious, but I'm not seeing it.